sdk/include/wdf/kmdf/1.9/wdfpool.h

```diff
@@ -0,0 +1,35 @@
+/*
+ * wdfpool.h
+ *
+ * Windows Driver Framework - Pool API's
+ *
+ * This file is part of the ReactOS wdf package.
+ *
+ * Contributors:
+ *   Created by Benjamin Aerni <admin@bennottelling.com>
+ *
+ * Intended Usecase:
+ *   Kernel mode drivers
+ *
+ * THIS SOFTWARE IS NOT COPYRIGHTED
+ *
+ * This source code is offered for use in the public domain. You may
+ * use, modify or distribute it freely.
+ *
+ * This code is distributed in the hope that it will be useful but
+ * WITHOUT ANY WARRANTY. ALL WARRANTIES, EXPRESS OR IMPLIED ARE HEREBY
+ * DISCLAIMED. This includes but is not limited to warranties of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ */
+#ifndef _WDFPOOL_H_
+#define _WDFPOOL_H_
+
+
+#if ((NTDDI_VERSION >= NTDDI_WIN2K)
```

Comment:
No need for this check since there's nothing inside.

Reply:
This is actually what my version of the headers did, lol

Reply:
Mine, too. But unlike the `#define _WDFPOOL_H_` it has no effect, so there's no need to replicate it.